src-electron/generator/helper-attribute.js

```diff
@@ -65,6 +66,77 @@ async function attributeDefault(options) {
   return templateUtil.templatePromise(this.global, p)
 }
 
+/**
+ * If helper that checks if an attribute is not supported.
+ * An attribute is not supported when it is a struct. However it is supported
```

Comment:
"supported" in what sense?  Matter absolutely supports struct-typed attributes....  This needs a better description of what it's really doing and why.

Reply:
If that's what we are trying to represent, I suggest we name it "if_attribute_type_unsupported_in_java_codegen" or something, so it's clear what it _really_ means.  And then we can put it in some helper js file that is _only_ pulled in by the java codegen, for clarity.

Reply:
It is now only unsupported if language is java. Else it is supported.